packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason-form.jsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { Formik, Field } from 'formik';
+import { Checkbox, Input, FormSubmitButton, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { PlatformContext } from '@deriv/shared';
+
+const initial_form = {
+    'I have other financial priorities': false,
```

Comment:
localize?

Reply:
These are just key values and we do not need localize for our keys. Values are translated already.